### PR TITLE
edit pass on Ident solutions /5

### LINKED ISSUE #29644: Style guide: Identity management solutions for .NET web app
Apply style guide

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: f6e45d58-5c8e-18b3-9a2e-86a65cedc74c
* Version Independent ID: f6e45d58-5c8e-18b3-9a2e-86a65cedc74c
* Content: [Identity management solutions for .NET web apps](https://learn.microsoft.com/en-us/aspnet/core/security/identity-management-solutions?view=aspnetcore-7.0)
* Content Source: [aspnetcore/security/identity-management-solutions.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/identity-management-solutions.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @JeremyLikness
* Microsoft Alias: **jeliknes**

---
[Associated WorkItem - 118040](https://dev.azure.com/msft-skilling/Content/_workitems/edit/118040)

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #29644

Per [MS style guide](https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice#use-bigger-ideas-fewer-words)

**_Use bigger ideas, fewer words_**
Our modern design hinges on crisp minimalism. Shorter is **_always_** better

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/security/identity-management-solutions.md](https://github.com/dotnet/AspNetCore.Docs/blob/2844bc5f02972b05a4ec97ef51771cd06114f602/aspnetcore/security/identity-management-solutions.md) | [Identity management solutions for .NET web apps](https://review.learn.microsoft.com/en-us/aspnet/core/security/identity-management-solutions?branch=pr-en-us-29791) |

### CODE REVIEW

File: aspnetcore/security/identity-management-solutions.md [hunk 1]
```diff
@@ -3,17 +3,20 @@ title: Identity management solutions for .NET web apps
...
+The following table provides an overview of various identity management solutions that can be used in ASP.NET Core apps. These solutions offer features and capabilities to manage [user authentication](xref:security/authentication/index), [authorization](xref:security/authorization/introduction), and [user identity](xref:security/authentication/identity) within an app. It includes options for apps that are:
 
- The table lists both open source and commercial solutions in alphabetical order, each accompanied by essential details such as license type, website, and documentation that is specific to ASP.NET Core integration. Explore the table below to discover the identity management solutions that best align with your application needs.
+* Container-based
+* Self-hosted, where you manage the installation and infrastructure to support it.
+* Managed, such as cloud-based or *Identity as a Service*, such as [IdentityServer4](https://identityserver4.readthedocs.io/en/latest/)
```
Comment (Rick-Anderson) [inline aspnetcore/security/identity-management-solutions.md:15]: What's `identity-as-a-service`? For sure that won't MT (machine translate) and I don't know what it is so I took a guess it's like IdentityServer4

Comment (tdykstra) [reply aspnetcore/security/identity-management-solutions.md:15]: "such as" twice in a sentence sounds awkward. I suggest 
```
Managed, also known as cloud-based or *Identity as a Service*.
```

**Review by tdykstra — APPROVED** [review]
LGTM, one suggestion.

File: aspnetcore/security/identity-management-solutions.md (hunk 1 above)
Comment (JeremyLikness) [inline aspnetcore/security/identity-management-solutions.md]: IdentityServer4 is actually self-hosted. Managed is what Microsoft Entra, Auth0, Okta, etc. are. This example won't work, but we could simplify by removing "identity as a service" and simply do:

"Managed, such as cloud-based services like [Microsoft Entra](https://learn.microsoft.com/aspnet/core/security/authentication/azure-active-directory/)"

**Review by JeremyLikness — CHANGES_REQUESTED** [review]

File: aspnetcore/security/identity-management-solutions.md (hunk 1 above)
Comment (tdykstra) [reply aspnetcore/security/identity-management-solutions.md]: If we use the name Entra here, we should use it in the table as well.

Comment (JeremyLikness) [reply aspnetcore/security/identity-management-solutions.md]: Sure. The table was written before the name change but should all be consistent. Thanks for your help with this @Rick-Anderson and @tdykstra.

**Comment by Rick-Anderson** [comment]
@JeremyLikness  what about moving the bare, ugly, nearly meaningless URL to the links in the name column? The table looks much better that way on a phone or tablet.

image

**Review by JeremyLikness — APPROVED** [review]
nice!

**Pull request merged**